apps/dotcom/src/components/IFrameProtector.tsx

```diff
@@ -26,8 +26,28 @@ and we should show an annoying messsage.
 If we're not in an iframe, we don't need to do anything.
 */
 
+export enum ROOM_CONTEXT {
+	PUBLIC_MULTIPLAYER = 'public-multiplayer',
+	PUBLIC_READONLY = 'public-readonly',
+	PUBLIC_SNAPSHOT = 'public-snapshot',
+	HISTORY_SNAPSHOT = 'history-snapshot',
+	HISTORY = 'history',
+	LOCAL = 'local',
+}
+
+enum EMBEDDED_STATE {
+	IFRAME_UNKNOWN = 'iframe-unknown',
+	IFRAME_NOT_ALLOWED = 'iframe-not-allowed',
+	NOT_IFRAME = 'not-iframe',
+	IFRAME_OK = 'iframe-ok',
+}
```

Comment:
```suggestion
const EMBEDDED_STATE = {
	IFRAME_UNKNOWN: 'iframe-unknown',
	IFRAME_NOT_ALLOWED: 'iframe-not-allowed',
	NOT_IFRAME: 'not-iframe',
	IFRAME_OK: 'iframe-ok',
} as const
```

Let's use this format for the enums. Enums are kind of a cursed TypeScript feature and so we've settled on this instead.